Pipfile

```diff
@@ -1,11 +1,9 @@
 [dev-packages]
-
+freezegun = "*"
```

Comment:
Can you lock the `Pipfile`?

Reply:
You are probably asking to perform `pipenv lock --dev` and committing the updated `Pipfile.lock` file.

I can do it, but I am still not familiar with proper usage of `pipenv`. If I remember well, [`pipenv` docs](https://docs.pipenv.org/basics/#general-recommendations-version-control) does not recommend committing `Pipenv.lock` in case, the program is supposed to be run under different python versions.

Reply:
> If I remember well, pipenv docs does not recommend committing Pipenv.lock in case, the program is supposed to be run under different python versions.

That's right. I'll have a look at it. It was committed prior to this, so let's keep it at least consistent.